app.css

```diff
@@ -0,0 +1,193 @@
+    --sclear-color: #00C4AC;
+    --snumber-color: #1C1C1C;
+    --stext-color: #ffffff;
+    --soperator-color: #89C0FF;
+}
+
+.dark-theme{
+    --primary-color: #0D0D0D;
+    --secondary-color: #FFFFFF;
+
+    --presult-color: #1C1C1C;
+    --sresult-color: #E1EFFF;
+
+    --phistory-color:#D8D8D8;
+    --poutput-color:#747474;
+
+    --shistory-color:#272727;
+    --soutput-color:#747474;
+
+    --sclear-color:#00DEC3;
+    --pclear-color: #00C4AC;
+
+    --pnumber-color: #1C1C1C;
+    --snumber-color: #E1EFFF;
+
+    --ptext-color: #ffffff;
+    --stext-color: #000000;
+}
+
+#icon{
+    float: right;
+    margin-right: 100px;
+    padding-top: 20px;
+    width: 30px;
+    height: 30px;
+    cursor: pointer;
+}
+
+/* ========================== */
+/*      iPhone13ProMax 1      */
+/* ========================== */
+
+#iPhone13ProMax1{
+    width: 428px;
+    height: 926px;
+    background: var(--primary-color);
+    margin: 0 auto;
+    border-radius: 50px;
+    position: relative;
+    top:20px;
+}
+
+#result{
+    width: 428px;
+    height: 332px;
+    background: var(--presult-color);
+    border-radius: 50px 50px 50px 50px;
+}
+
+#history{
+    text-align: right;
+    height: 94px;
+    margin: 0 20px;
+    padding-top: 20px;
+    padding-bottom: 30px;
+
+    font-family: 'Work Sans';
+    font-style: normal;
+    font-weight: 500;
+    font-size: 80px;
+    line-height: 94px;
+    letter-spacing: 0.1em;
+
+    color: var(--phistory-color);
+}
+
+#output{
+    text-align: right;
+    height: 70px;
+    margin: 10px 20px;
+
+    font-family: 'Work Sans';
+    font-style: normal;
+    font-weight: 600;
+    font-size: 60px;
+    line-height: 70px;
+    letter-spacing: 0.1em;
+    
+    color: var(--poutput-color);
+
+}
+
+#keyboard{
+    height: 594px;
+    padding-top: 50px;
+    padding-left: 12px;
+}
+
+
+.operator, .number {
+   
+    width: 80px;
+    height: 80px;
+    margin: 10px;
+    float: left;
+    border-radius: 30px;
+    border-width: 0;
+    border-style: none;
+    font-weight: bold;
+    font-size: 25px;
+    
+}
+
+#clear{
+    width: 180px;
+    height: 80px;
+   
+    background: var(--pclear-color);
+    border-radius: 30px;
+    border-style: none;
+
+    font-weight: 150;
+    font-size: 25px;
+    color: var(--ptext-color);
+}
+
+.number, .operator{
+    cursor: pointer;
+}
+
+.operator:active, .number:active{
+    font-size: 15px;
+}
+
+.operator:focus, .number:focus {
+    outline: 0;
+} 
+
+.number{
+    background:  var(--pnumber-color);
+    color: var(--ptext-color);
+    border-radius: 50%;
+}
+
+.operator{
+    background:  var(--poperator-color);
+    border-radius: 30px;
+    border-style: none;
+    font-weight: bold;
+    font-size: 25px;
+    color: var(--ptext-color);
+}
+
+.ce{
+    background:  var(--pnumber-color);
+    color: var(--ptext-color);
+    border-radius: 50%;
+}
+
+.dot{
+    background:  var(--pnumber-color);
+    color: var(--ptext-color);
+    border-radius: 50%;
+}
```

Comment:
we are missing end of file here. Please check.

Reply:
That's the End of CSS file Sahil